.github/workflows/ci.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       - name: ruff
         if: matrix.run_lint == true
         run: |
-          ruff .
+          ruff check .
```

Comment:
We do not mix CI changes with other changes, simply because changing CI requires a bit more than just a PR.

Reply:
Hmm, tests don't pass without this change, btw I also added this to the dev deps bump PR in #2241 since it is needed there as well I think for tests to pass.

Reply:
https://github.com/pymodbus-dev/pymodbus/pull/2241#issuecomment-2239907728